src/entities/Proposal/model.ts

```diff
@@ -440,4 +459,12 @@ export default class ProposalModel extends Model<ProposalAttributes> {
       SQL` || `
     )})`
   }
+
+  static async getOpenPitchsTotal() {
+    const query = SQL`
+    SELECT COUNT(DISTINCT (configuration->>'linked_proposal_id')) AS total
+    FROM ${table(this)} WHERE type = ${ProposalType.Tender} AND status = ${ProposalStatus.Pending};`
```

Comment:
Shouldn't this be a count of pending pitch proposals?

Reply:
This returns the pitches count where tender are being submitted aka tender window open